Report.java

```diff
@@ -201,5 +204,61 @@ private void printSpriteCounts(PrintWriter printW, int i)
                 + sprites[j].getSoundBlocksForSprite());
             printW.println();
         }
-    }   
+    }
+
+    /**
+     * Print total variable usage counts.
+     *
+     * @param printW - the printWriter to use
+     * @param i - the submission to get count for
+     */
+    public void printTotalVariableUsage(PrintWriter printW, int i)
+    {
+        String[] vars = submissions[i].getGlobalVariables();
+        for (int j = 0; j < vars.length; j++)
+        {
+            printW.println("\t" + vars[j]
+                + ": " + submissions[i].getProgramVariableUsageCount(vars[j]));
```

Comment:
I think we need to somehow make it clear we are listing the variable name and how many times it is used. Just listing the variable and the number it is not very clear what information we are providing.